spec/integration/user_post_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe 'User Integration', type: :system do
+  before do
+    @user1 = User.create(id: 100, name: 'Username 1', photo: 'test1.jpg', postcounter: 10, bio: 'User 1 bio')
+    @user2 = User.create(id: 201, name: 'Username 2', photo: 'test2.jpg', postcounter: 20)
+  end
+
+  it 'redirect to the post show page' do
+    post1 = Post.create(id: 103, author: @user1, title: 'Post title 1 by User 1',
```

Comment:
- [ ] Please kindly implement the highlighted tests cases below 👇  that are stated in the requirement. 👍 

<img width="668" alt="Untitled" src="https://github.com/devalibello/blog-app/assets/103386183/d561b90b-f039-42fc-a14e-470f001ac1ca">